iox_catalog/src/interface.rs

```diff
@@ -1061,13 +1116,15 @@ pub(crate) mod test_helpers {
         assert_eq!(NEW_COLUMN_LIMIT, modified.max_columns_per_table);
 
         const NEW_RETENTION_PERIOD: i64 = 5;
+        const NEW_RETENTION_PERIOD_NS: i64 = 5 * 60 * 60 * 1000 * 1000 * 1000;
```

Comment:
Perhaps we ensure this stays in sync:

```suggestion
        const NEW_RETENTION_PERIOD_NS: i64 = NEW_RETENTION_PERIOD * 60 * 60 * 1000 * 1000 * 1000;
```